### PR TITLE
Add linebreaks in Promise doc example log output

### PR DESCRIPTION
**Pull request by jadjoubran**
Follow up for #1100

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/promise/index.html [hunk 1]
```diff
@@ -333,11 +333,11 @@ <h3 id="Advanced_Example">Advanced Example</h3>
   let thisPromiseCount = ++promiseCount;
   let log = document.getElementById('log');
   // begin
-  log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Started');
+  log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Started<br />');
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/promise/index.html:336]:
```suggestion
  log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Started<' + 'br>');
```

File: files/en-us/web/javascript/reference/global_objects/promise/index.html [hunk 2]
```diff
@@ -333,11 +333,11 @@ <h3 id="Advanced_Example">Advanced Example</h3>
...
   // We make a new promise: we promise a numeric count of this promise, starting from 1 (after waiting 3s)
   let p1 = new Promise((resolve, reject) =&gt; {
     // The executor function is called with the ability to resolve or reject the promise
-    log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Promise constructor');
+    log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Promise constructor<br />');
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/promise/index.html:340]:
```suggestion
    log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Promise constructor<' + 'br>');
```

File: files/en-us/web/javascript/reference/global_objects/promise/index.html [hunk 3]
```diff
@@ -349,13 +349,13 @@ <h3 id="Advanced_Example">Advanced Example</h3>
   // and what to do when the promise is rejected with the catch() call
   p1.then(function(val) {
     // Log the fulfillment value
-    log.insertAdjacentHTML('beforeend', val + ') Promise fulfilled');
+    log.insertAdjacentHTML('beforeend', val + ') Promise fulfilled<br />');
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/promise/index.html:352]:
```suggestion
    log.insertAdjacentHTML('beforeend', val + ') Promise fulfilled<' + 'br>');
```

File: files/en-us/web/javascript/reference/global_objects/promise/index.html [hunk 4]
```diff
@@ -349,13 +349,13 @@ <h3 id="Advanced_Example">Advanced Example</h3>
...
   }).catch((reason) =&gt; {
     // Log the rejection reason
     console.log(`Handle rejected promise (${reason}) here.`);
   });
   // end
-  log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Promise made');
+  log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Promise made<br />');
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/promise/index.html:358]:
```suggestion
  log.insertAdjacentHTML('beforeend', thisPromiseCount + ') Promise made<' + 'br>');
```

**Review by sideshowbarker — DISMISSED** [review]
What you had won’t work — those `<br />` tags just get dropped on the floor.

And it’s not necessary are useful to do `<br />` rather than just `<br>`; we’re not using XML.

**Review by sideshowbarker — CHANGES_REQUESTED** [review]
What you had won’t work — those `<br />` tags just get dropped on the floor.

And it’s not necessary are useful to do `<br />` rather than just `<br>`; we’re not using XML.

**Comment by jadjoubran** [comment]
Got it 👍

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**